lib/logstash/inputs/stomp.rb

```diff
@@ -59,6 +75,10 @@ def register
       connect
       subscription_handler # is required for re-subscribing to the destination
     }
+
+    # Display configuration information
+    @logger.info("Reconnect is disabled, plugin will not reconnect on connection error") if !@reconnect
+    @logger.info("Debug is disabled, turn on debug for full exceptions") if !@debug
```

Comment:
Aah I see. I'm relatively new to the Elastic stack. This explains why I wasn't seeing the errors when I was running a command line of the plugin. What is the correct way to displays info messages generated by a plugin? Thanks!